go.mod

```diff
@@ -256,7 +256,7 @@ require (
 )
 
 replace (
-	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.28.0-sdk-v0.46.16
+	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.28.0-sdk-v0.46.16.0.20250331072412-0673d7c9b833
```

Comment:
once v3.x tax is built with this version of the sdk, it should be fully possible to perform a signalling upgrade from an embedded v3 to a native v4